danielMunteanu/src/main/java/Item.java

```diff
@@ -0,0 +1,66 @@
+public class Item {
+    public long id;
```

Comment:
can make it private. as you want

Reply:
is public because in the task says like this

Reply:
in the task is not specified to create setters and getters. They are used in order to access private data. In your case, you have public data and getters+ setters